core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -161,6 +161,24 @@ public void hadoopShouldLoadFileSystemWhenConfiguredTest() throws Exception {
   }
 
   /**
+  * Ensures that when ListStatusNonExisting should Throw FileNotFoundException.
+  */
+  @Test
+  public void shouldThrowFileNotFoundExceptionWhenListStatusNonExisting() throws Exception {
```

Comment:
We typically end our test names with `Test`. The "should" is implicit in all tests, so we don't need to include it as part of the test name.